easybuild/easyconfigs/l/libdierckx/libdierckx-1993-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'libdierckx'
+version = '1993'
+
+homepage = 'http://www.netlib.org/dierckx'
+description = """ Fortran subroutines for calculating smoothing splines for various
+ kinds of data and geometries, with automatic knot selection"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

Comment:
Please use the `GCC` toolchain for this. Making that change will mean the filename also needs updating and the builddependency on `binutils` should be removed.

Reply:
Hi @branfosj.  This module provides a static library that I use with both Intel and FOSS toolchains.  (It doesn't provide any toolchain-specific `*.mod` files.)  If I change to explicitly using the `GCC` toolchain then I'll have to also create an `intel` version whereas at the moment I can use `GCCcore` for both.  Is this an acceptable reason for keeping this EB file associated with the `GCCcore` toolchain or is there some other reason you're suggesting changing?  Thanks!